src/content/en/tools/chrome-devtools/network-performance/resource-loading.md

```diff
@@ -1,22 +1,27 @@
 project_path: /web/_project.yaml
 book_path: /web/tools/_book.yaml
-description: Measure the network performance of your web application  using the Chrome DevTools Network panel.
+description: Measure the network performance of your web application using the Chrome DevTools Network panel.
 
-{# wf_updated_on: 2016-02-21 #}
+{# wf_updated_on: 2016-12-28 #}
 {# wf_published_on: 2015-04-13 #}
 
 # Measure Resource Loading Times {: .page-title }
```

Comment:
See [this note](https://github.com/google/WebFundamentals/pull/4000/files#diff-b05a1a6b90e5fd87f39d5e61eb51137a).